stix_shifter/stix_transmission/src/modules/aws_cloud_watch_logs/aws_cloud_watch_logs_results_connector.py

```diff
@@ -0,0 +1,146 @@
+from ..base.base_results_connector import BaseResultsConnector
+import json
+from flatten_json import flatten
+import copy
+from .....utils.error_response import ErrorResponder
+from os import path
+
+MAX_LIMIT = 1000
+
+
+class InvalidParameterException(Exception):
+    pass
+
+
+class AWSCloudWatchLogsResultsConnector(BaseResultsConnector):
+    def __init__(self, client):
+        self.client = client
+
+    def create_results_connection(self, search_id, offset, length):
+        """
+        Fetching the results using search id, offset and length
+        :param search_id: str, search id generated in transmit query
+        :param offset: str, offset value
+        :param length: str, length value
+        :return: dict
+        """
+        return_obj = dict()
+        response_dict = dict()
+        try:
+            query = dict()
+            offset = int(offset)
+            length = int(length)
+            total_records = offset+length
+            if total_records <= MAX_LIMIT:
+                query['queryId'] = search_id
+                response_dict = self.client.get_query_results(**query)
+                return_obj['success'] = True
+                results = response_dict['results'][offset:total_records]
+                result_list = []
+                guardduty_common_attr = ["account", "source", "detail_region", "detail_id",
```

Comment:
`guardduty_common_attr` can this list be declared global? its a static list and dont see any update in the list. its just passed one function to another.

Reply:
OR i wonder if it can be put inside json/ folder as a json file to make it more organized and clean. any aws service that needs common attributes can have their own json file or we can put all of them in a single `common_attributes.json` file. key would the aws service name and value would the the list of attributes:
`{"vpc": [...], "guarduty": [...]}`

Reply:
done